src/ImageSharp.Web/Middleware/ImageSharpMiddleware.cs

```diff
@@ -492,9 +492,9 @@ private ValueTask StreamDisposeAsync(Stream stream)
             }
         }
 
-        private static string GetUri(HttpContext context, IDictionary<string, string> commands)
+        private static string GetUri(HttpContext context, IDictionary<string, string> commands, bool ignoreHost)
```

Comment:
Quick note that this code can be much improved. We know the size of all strings, so we can allocate the right buffer size instead of growing it in the SB.
Also using the Format("{0}/") is too much for a simple concatenation.
Only `QueryString.Create(commands)` needs to be done before to know what will be the size.
Should I file an issue or not worth it?

Reply:
Yep, that's a valid point. Please file an issue. I'm up for any optimizations we can through at the library.